docs/tutorial.rst

```diff
@@ -37,7 +37,10 @@ Running ``stdpopsim`` with the command-line interface (CLI)
 ***********************************************************
 
 In order to use the ``stdpopsim`` CLI the ``stdpopsim`` package must be
-installed (see :ref:`Installation <sec_installation>`). The CLI provides access
+installed (see :ref:`Installation <sec_installation>`).
```

Comment:
Maybe better to put a quick note at the top of the Installation section, so that we don't have to say the same thing lots of times? Less likely to go out of date then.

Reply:
Do you think add it as a note to installation section instead of putting in the intro and tutorial?

Reply:
Yes - that way when someone click on "Installation" the first thing they see is "Oh, I don't actually need to install at all; cool!"

So, we don't change the intro and tutorial at all, just the installation section. Or the intro and tutorial link to a section somewhere else which tells them about the binder. I just don't want to put the same link into the docs n times - links go out of date and it's much easier to update them if there's only one place!